File: mario.c
#include <stdio.h>

int main(void)
{
    //input
    int height;
    do
    {
        printf("Height: ");
        scanf("%d", &height);
    }
    while (height < 1 || height > 8);


    //print
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        for (int x = 1; x <= i; x++)
        {
            printf("#");
        }
        printf("  ");
        for (int y = 1; y <= i; y++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int
#input
height = get_int("Height: ")
while height < 1 or height > 8:
    height = get_int("Height: ")


#print
for i in range(1, height + 1):
    for j in range (0, height - i):
        print(" ", end="")
    for x in range(1, i + 1):
        print("#", end="")
    print("  ", end="")
    for y in range(1, i + 1):
        print("#", end="")
    print("\n", end="")
